documentation/src/main/asciidoc/topics/proc_download_hotrod_java_client.adoc

```diff
@@ -1,13 +1,9 @@
 [id='download_java_client']
-= Downloading the Hot Rod Java Client
-{brandname} provides a reference Hot Rod Java client implementation.
+= Getting the Hot Rod Java Client
 
-//Community
-ifndef::productized[]
-Visit the link:http://infinispan.org/hotrod-clients[Hot Rod clients] page on _infinispan.org_ to download the Java client and find client implementations in other languages.
-endif::productized[]
+Add the `infinispan-client-hotrod` dependency to your `pom.xml` as follows:
 
-//Product
-ifdef::productized[]
-Visit the link:{download_url}[{brandname} downloads] page on the {portal} to download the Java client and find client implementations in other languages.
-endif::productized[]
+[source,xml]
+----
+include::dependencies_maven/hotrod_java_client.xml[]
```

Comment:
this path doesn't exist:
```
Unresolved directive in ../../topics/proc_download_hotrod_java_client.adoc - include::dependencies_maven/hotrod_java_client.xml[]
```

Reply:
oh, man! thanks for catching that @pruivo